packages/server/index.js

```diff
@@ -2,6 +2,13 @@ const { initializeStartTime } = require('./lib/util/performance_benchmark')
 
 const run = async () => {
   initializeStartTime()
+
+  const hook = './hook'
+
+  if (require.name !== 'customRequire') {
+    require(`${hook}-require`)
```

Comment:
nit: inline the constant `hook` value?

Reply:
[cypress-io/cypress@`ec34a53` (#23732)](https://github.com/cypress-io/cypress/pull/23732/commits/ec34a537a4879edcfbc0409b0f81f16a94e97ee9)